solution-base/deps.txt

```diff
@@ -0,0 +1,6 @@
+kubedb/operator:v0.13.0-rc.0
+kubedb/redis:5.0.3-v1
```

Comment:
probably my mistake from being unclear in a previous comment, but missing the `redis_exporter` image  also needed by `kubedb`